site/about/glossary/key_concepts/_test-suite.qmd

```diff
@@ -0,0 +1,4 @@
+test suite
+: A collection of tests which are run together to generate model documentation end-to-end for specific use-cases. 
+
+  - e.g. The [`classifier_full_suite`](../../validmind/validmind/test_suites/classifier.html#ClassifierFullSuite) test suite runs tests from the [`tabular_dataset`](../../validmind/validmind/test_suites/tabular_datasets.html) and [`classifier`](../../validmind/validmind/test_suites/classifier.html) test suites to fully document the data and model sections for binary classification model use-cases.
```

Comment:
```suggestion
  - For example, the [`classifier_full_suite`](../../validmind/validmind/test_suites/classifier.html#ClassifierFullSuite) test suite runs tests from the [`tabular_dataset`](../../validmind/validmind/test_suites/tabular_datasets.html) and [`classifier`](../../validmind/validmind/test_suites/classifier.html) test suites to fully document the data and model sections for binary classification model use-cases.
```

Reply:
Why do we prefer the "For example" here over the shorthand?

Reply:
It's pretty typical for style guides to spell out words rather than to use Latin abbreviations — easier to understand for more people (and also looks cleaner). For example: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

Reply:
Makes sense!